semantic_model_generator/snowflake_utils/snowflake_connector.py

```diff
@@ -83,18 +138,29 @@ def get_table_representation(
     columns_df: pd.DataFrame,
     max_workers: int,
 ) -> Table:
-    table_comment = columns_df[_TABLE_COMMENT_COL].iloc[0]
+    table_comment, is_auto_generated = _get_table_comment(conn, table_name, columns_df)
+
+    def _get_ndv_per_column(column_row: pd.Series, ndv_per_column: int) -> int:
```

Comment:
dumb q - what's `ndv`? number of distinct values?

Reply:
yes, number of distinct values!